subcommand/build/main.go

```diff
@@ -24,12 +25,21 @@ import (
 	"github.com/bytearena/core/common/utils"
 )
 
-const (
+var (
 	DOCKER_BUILD_FILE = "Dockerfile"
 	SHOW_USAGE        = true
 	DONT_SHOW_USAGE   = false
+
+	WATCH_DIR_RECURSION_DEPTH = uint(100)
+	WATCH_IGNORE_DIRS         = map[string]bool{
+		".git": true,
```

Comment:
On my system `git` is creating and removing some lock file at each command, we want to avoid triggering a build.

Should we ignore every dotfiles? Some are useful to the user (`.env`), but some are not (compilation caches).